x-pack/test/functional/page_objects/lens_page.ts

```diff
@@ -433,17 +433,21 @@ export function LensPageProvider({ getService, getPageObjects }: FtrProviderCont
       metaKey?: 'shift' | 'alt' | 'ctrl'
     ) {
       const elements = await find.allByCssSelector(
-        `[data-test-subj="${group}"]  [data-test-subj="lnsDragDrop-keyboardHandler"]`
+        `[data-test-subj="${group}"] [data-test-subj="lnsDragDrop-keyboardHandler"]`
       );
       const el = elements[index];
       await el.focus();
       await browser.pressKeys(browser.keys.ENTER);
       for (let i = 0; i < steps; i++) {
+        // This needs to be slowed down to avoid flakiness
+        await common.sleep(200);
```

Comment:
Ugh these calls produce a ton of flakiness because they are running too fast. I had the same issue with `lensPage.setPalette` function in this file.

The issue was seen separately in the following:
- `dimensionKeyboardDragDrop` ([failed tests](https://buildkite.com/elastic/kibana-flaky-test-suite-runner/builds/8081))
- `dimensionKeyboardReorder` ([failed tests](https://buildkite.com/elastic/kibana-flaky-test-suite-runner/builds/8082))

After adding the `sleep(200)` in the functions fixed all the issues https://buildkite.com/elastic/kibana-flaky-test-suite-runner/builds/8083.